src/commands/org/open.ts

```diff
@@ -115,7 +117,8 @@ export class OrgOpenCommand extends SfCommand<OrgOpenOutput> {
       if (err instanceof Error) {
         if (err.message.includes('timeout')) {
           const domain = `https://${/https?:\/\/([^.]*)/.exec(url)?.[1]}.lightning.force.com`;
-          const timeout = new Duration(new Env().getNumber('SFDX_DOMAIN_RETRY', 240), Duration.Unit.SECONDS);
+          const domainRetryTimeout = env.getNumber('SF_DOMAIN_RETRY') ?? env.getNumber('SFDX_DOMAIN_RETRY', 240);
```

Comment:
should this be the same `||`?

Reply:
No because `env.getNumber()` does not have a default value (like getBoolean() does) so if the env var is not set it returns undefined.